cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -15,29 +15,29 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 service PackagesService {
   rpc GetAvailablePackageSummaries(GetAvailablePackageSummariesRequest) returns (GetAvailablePackageSummariesResponse) {
      option (google.api.http) = {
-       get: "/core/packages/v1alpha1/availablepackagesummaries"
+       get: "/core/packages/v1alpha1/availablepackages"
```

Comment:
Shouldn't this URL also be cluster/ns-dependant? Or we are just gonna encode them via query params?
I mean, it's okay in that the rest of the URIs have the `c` and `ns` under the `plugin`, so there isn't any semantic issue.
Besides, they are optional, so not passing as query params would imply multiplexing the endpoint in: `.../availablepackages`, `.../availablepackages/c/<cluster>` and `.../availablepackages/c/<cluster>/ns/<ns> and I don't know if it is possible.

Reply:
Yes, for the list endpoints I've left them as qparams since they are optional. The only way I see for multiplexing different URLs for the same functionality is by using different RPC function names (ie. `GetAvailablePackageSummariesPerNamespace` with its own get, with the grpc function just calling the other one... not pretty).